cancat/uds.py

```diff
@@ -366,7 +366,7 @@ def ScanDIDs(self, start=0, end=0x10000, delay=0):
     def SecurityAccess(self, level, key):
         msg = self._do_Function(SVC_SECURITY_ACCESS, subfunc=level, service = 0x67)
         if msg is None:
-            return "\x00\x7f\x00\x35"
```

Comment:
I really have no clue what this was - something vestigial from testing probably. Was wreaking all kinds of havoc when trying to write some code that used this functionality